### ISSUE TITLE
Request

### ISSUE THREAD

**Issue by OrionRandD** [briano1905/typings#61]
What about RLL?
Hebrew and Arabic?
Any plans to add them?
Thx a lot

**Comment by jakergrossman** [comment]
I think I have it working, but I wanted to verify my word list before I submit the pull request. I don't speak Arabic, so I don't know if these words are any good. I've attached my word file below, it was from a stopwords file. If these words will not suffice, would you mind helping me find suitable words?

[arabic.txt](https://github.com/briano1905/typings/files/4131912/arabic.txt)

**Comment by OrionRandD** [comment]
I have contacted someone who know Arabic well. I hope that he will pop up here to help us with this issue...
Thx for replying

**Comment by mohataher** [comment]
Hi guys, I'm happy to help. But the issue's title and description is not comprehensive enough to make a good judgment here.

@jakergrossman your list looks like it's copied from [arabic_stop_words](https://github.com/mohataher/arabic-stop-words) repo?

**Comment by jakergrossman** [comment]
@mohataher, that is correct. Essentially, I need a list of the most common Arabic words.

**Comment by mohataher** [comment]
Arabic stop words are not essentially the most common words in the language. Let me have a better look at your application and get back to you by Sunday.

**Comment by jakergrossman** [comment]
Sounds good! I really just used those to verify formatting, thanks for helping! Eager to hear back from you.

**Comment by mohataher** [comment]
Here you go guys! I have filtered some of the Arabic [diacritics](https://en.wikipedia.org/wiki/Arabic_diacritics) because it makes it complicated to type words. I have also removed Arabic  punctuations because they're irrelevant here. This list is not ranked according to the word popularity but alphabetically though. 

[arabic.txt](https://github.com/briano1905/typings/files/4176868/arabic.txt)

Let me know if you guys need any further filtering!

**Comment by jakergrossman** [comment]
Thanks so much! That’s exactly what we needed! Much appreciated.
